File: Programmers/JAVA/lv1/lv1_문자열 내 마음대로 정렬하기.java
https://programmers.co.kr/learn/courses/30/lessons/12915

import java.util.*;

class Solution {
    public String[] solution(String[] strings, int n) {
        String[] answer = new String[strings.length];
        Arrays.sort(strings);
        
        int index = 0;
        for(int i = 97; i <= 122; i++){					//97~122:a~z, 65~90:A~Z
            for(int j = 0; j < strings.length; j++){
                if(strings[j].charAt(n) == (char)i){
                    answer[index] = strings[j];
                    index++;
                }   
            }    
        }
        
        return answer;
    }
}

File: Programmers/C#/lv1/lv1_문자열 내 마음대로 정렬하기.cs
https://programmers.co.kr/learn/courses/30/lessons/12915

using System.Linq;

public class Solution
{
    public string[] solution(string[] strings, int n)
    {
        string[] answer = new string[] { };

        answer = strings.OrderBy(x => x).OrderBy(x => x[n]).ToArray();

        return answer;
    }
}